doc/funding/priorities.md

```diff
@@ -0,0 +1,68 @@
+2. **Improve the efficiency of our contributors.** It takes less effort to maintain and develop technology in the Jupyter ecosystem. This includes reducing toil, automating common actions, defining processes and responsibility that helps us execute more reliably and effectively, etc.  
+3. **Facilitate learning across the Jupyter subprojects.** Currently many subprojects act as independent and self-contained teams. They do not have a reliable way for learning from others in the Jupyter ecosystem, sharing resources, and sharing skills. Creating mechanisms for connecting these separate parts of the Jupyter ecosystem will result in more coherent and efficient actions across its community.
+
+### Examples of how to grow contributor capacity
+
+- Hold events that grow Jupyter’s visibility and presence.  
+- Hold events that give newcomers an opportunity to collaborate with maintainers in real-time to gain comfort with participating in the project.  
+- Improve the documentation for navigating the Jupyter project as a whole (e.g., helping newcomers understand the project’s subprojects, skills needed to contribute to each, etc)  
+- Improve the documentation across each of Jupyter’s subprojects, with an emphasis on documentation for contributors.
+- Fund maintainer time to organize community meetings and coordinate their efforts, with an emphasis on making them accessible and useful to non-maintainers. (for example, a “Developer Experience” or “Community Manager” role)  
+- Fund regular, low-effort events that give others an opportunity to pair with maintainers in common maintainer actions like PR review, issue triage, etc.  
+- Fund maintainer time to steward contributions from external contributors and provide them feedback with the goal of increasing their comfort with contributing.  
+- Fund development work on maintainer infrastructure that reduces toil and grows efficiency (e.g., improving the Jupyter Releaser technology and documentation)
+
+## Improve the reliability, security, and consistency of Jupyter’s software
+
+Many of Jupyter’s repositories lack the capacity needed to be considered reliable, safe, and consistent. This is due to a combination of bandwidth constraints, tooling, training, and coordination. As a result, Jupyter’s subprojects often accrue technical debt that burns out their maintainers, and that increases the variability of their outputs. Member organizations have a desire for Jupyter’s technology to be *reliable, safe, and consistent*. This allows them to confidently build their own technical dependencies on Jupyter technology and use them in production contexts.
+
+By using Foundation funding to **improve the reliability, security, and consistency of Jupyter’s software and team practices**, we can help Jupyter community members more sustainably manage their technical and team debt, release more reliable software, and reduce the burden on user organizations that deploy this software internally.
+
+### Key outcomes
```

Comment:
This isn't actionable feedback for this PR, but it might be worthwhile for the technical subcommittee to think a bit more about these. I wonder if there are more. Not a blocker for this PR. Seems like a good starting point.

Reply:
Good point - I'll add a brief note that these are non-exhaustive. The main point is to help shape the kinds of outcomes we're talking about.